spirv_msl.cpp

```diff
@@ -3114,67 +3147,98 @@ void CompilerMSL::add_variable_to_interface_block(StorageClass storage, const st
 		else
 		{
 			bool masked_block = false;
+			uint32_t location = 0;
```

Comment:
The use of location here is kinda confusing. It's used as a sentinel *and* a valid value at the same time. I would suggest using a more clear sentinel like UINT32_MAX or something.

Reply:
Fixed.